packages/mds-compliance-engine/tests/engine/count-tests.spec.ts

```diff
@@ -119,22 +118,22 @@ describe('Tests Compliance Engine Count Functionality:', () => {
         speed: rangeRandomInt(10)
       })
 
-      test.assert(
+      expect(
         isCountRuleMatch(
           COUNT_POLICY.rules[0] as CountRule,
           GEOGRAPHIES,
           LAdevices[0],
           LAevents[0] as VehicleEvent & { telemetry: Telemetry }
         )
-      )
-      test.assert(
+      ).toBeTruthy()
```

Comment:
Do you prefer `.toBeTruthy()` to `.toEqual(true)`? Are there instances where a `boolean` is not returned?